docs/source/tutorials/datalab/datalab_quickstart.ipynb

```diff
@@ -682,6 +683,15 @@
     "examples_w_issue.head()"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lab._label_map"
```

Comment:
what is the point of this? we should not expose users to internal class mappings

Reply:
Sorry, rogue cell.

Reply:
Removed this cell.